gosafe.lic

```diff
@@ -0,0 +1,17 @@
+=begin
+=end
+
+custom_require.call(%w[common common-travel])
+
+class Gosafe

```

Comment:
💡 I'd prefer `GoSafe`, but not a big deal

Reply:
Changed to GoSafe

Reply:
Thank you 😄

Reply:
Ok, i think this is ready put in purpose in script documentation.